layers/+web-services/wakatime/README.org

```diff
@@ -62,6 +62,9 @@ Example:
                  ))
 #+END_SRC
 
+*** Note about api key and bug submission
```

Comment:
Your new headline should be mentioned in the toc. Also there are some typos in your text you should fix.
```suggestion
If you use the built in Spacemacs bug submission tool it will include all of your layer configs including your secret api key. Please remove it before submitting.

Even better don't store your key directly in the layer config. Instead put the following code in your user-config which is set after your layer config:
#+BEGIN_SRC emacs-lisp
  (setq wakatime-api-key "your-api-key")
#+END_SRC